client/config/config.go

```diff
@@ -91,6 +91,10 @@ type Config struct {
 	// dynamically. It should be given as Cores * MHz (2 Cores * 2 Ghz = 4000)
 	CpuCompute int
 
+	// MemoryMB is the default node total memory if it cannot be determined
```

Comment:
`MemoryMB is the default node total memory in megabytes if it..`, would rather make the documentation explicit though the variable name is a clue.